src/tests/miscellaneous/mock-adb/common-adb-configs.js

```diff
@@ -93,6 +94,16 @@ function addPortForwardingCommands(id, output, port) {
     };
 }
 
+function addInputKeyeventCommands(id, output) {
+    const keyeventCodes = [19, 20, 21, 22, 61, 66];
```

Comment:
I don't mind this being repeated here but it would be nice for this to make it more clear how the magic numbers were arrived at, something like:

```ts
// These are the cases required for the virtual keyboard in unified tab stops
// See https://android.documentation/keycodes
const keyeventCodeMap = {
    'left-arrow': 19,
    'right-arrow': 20,
    // ... etc
};

for (const keyeventCode of Object.values(keyeventCodeMap)) {
    // ...
}
```